### PR TITLE
Use Lando for Postgres.

### LINKED ISSUE #4212: Use Lando for Postgres
This should help with us having a bunch of different postgres versions, resulting in `rake db:migrate` changing files.

### PR DESCRIPTION
**Pull request by tpendragon**
This keeps us locked to the Postgres version we're running in
production, and keeps all the developers on the same version for this
project as well, resulting in a consistent schema file on rake
db:migrate.

Closes #4212

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,36 +71,43 @@ Remember you'll need to run `bundle install` and `yarn install` on an ongoing ba
...
-### Manual Setup
+You can either run Solr/Postgres locally or spin them up in Docker containers
+with Lando.
+
+### Lando
+
+Lando will automatically set up docker images for Solr and Postgres which match
+the versions we use in Production. The ports will not collide with any other
+projects you're using Solr/Postgres for, and you can easily clean up with `lando
+destroy` or turn off all services with `lando spindown`.
```
[the remark this answers is not in the document]
Comment (tpendragon) [reply README.md:82]: Ugh, it's `lando poweroff`. Thanks.

File: README.md [hunk 2]
```diff
@@ -71,36 +71,43 @@ Remember you'll need to run `bundle install` and `yarn install` on an ongoing ba
...
    - `RAILS_ENV=test rake db:setup`
-   - `rake figgy:test`
    - In a separate terminal: `bundle exec rspec`
    - Run jest tests: `yarn test`
-2. For development:
+1. For development:
    - ``export SECRET_KEY_BASE=`rake secret` ``
    - `rake db:setup`
-   - In a separate terminal: `foreman start`
+   - In a separate terminal: `foreman start` (you can close the one launching solr)
```
Comment (hackmastera) [inline README.md:94]: I'm not sure close the one launching solr is correct. Someone might have to comment it out. In general foreman doesn't start services in separate panes / windows / tabs.

Perhaps we should have a followup issue to update foreman to just run the web services, and documnt that the solr service will be separate in either case.

**Review by hackmastera — CHANGES_REQUESTED** [review]
Tried this out and it works for me. Just left another comment about the foreman situation.

File: README.md (hunk 2 above)
Comment (tpendragon) [reply README.md]: Updated it. Foreman already isn't running solr for Figgy.

**Review by hackmastera — APPROVED** [review]
👍 💯

**Pull request merged**